index.js

```diff
@@ -781,10 +781,10 @@ Command.prototype.optionMissingArgument = function(option, flag) {
  * @api private
  */
 
-Command.prototype.unknownOption = function(flag) {
+Command.prototype.unknownOption = function(unknown) {
```

Comment:
So, if the method now accepts array of unknown options, that it should be named as `unknownOptions` etc.